apps/api/scripts/test-slippage.js

```diff
@@ -0,0 +1,139 @@
+const fs = require('fs');
```

Comment:
this script should make it easier to test with real data we get the right slippage tolerance, and we can reproduce easily the calculations

I will use the output files to refine the algorithms defining a test that would ensure we get the right value for a realistic example of 288 prices